apisix/core/utils.lua

```diff
@@ -375,5 +377,28 @@ end
 -- Resolve {$1, $2, ...} in the given string
 _M.resolve_var_with_captures = resolve_var_with_captures
 
+local function server_func(func_name, body, headers)
+    local local_conf = config_local.local_conf()
+
+    if local_conf.apisix and local_conf.apisix.server_func_addr
+        and local_conf.apisix.server_func_addr ~= "" then
+        local addr = local_conf.apisix.server_func_addr
+        if not string.has_suffix(addr, "/") then
+            addr = addr .. "/"
+        end
+        log.info("external server func to: " .. addr .. func_name ..
+            ", body: " .. json.encode(body))
+        local httpc = http.new()
+        local res, err = httpc:request_uri(addr .. func_name, {
+            method = "POST",
```

Comment:
Would it be possible to replace this constant with a parameter? This would increase flexibility.

Reply:
Using parameters can indeed be more flexible.

https://github.com/apache/apisix/pull/10112#issuecomment-1716791638

As I mentioned before, a usage pattern is provided here. If users do not want to use this pattern, they can implement the logic by themselves.